plugins/chat/assets/javascripts/discourse/components/channels-list.js

```diff
@@ -13,6 +13,19 @@ export default class ChannelsList extends Component {
   @service session;
   @service currentUser;
 
+  @tracked hasScrollbar = false;
+
+  @action
+  computeHasScrollbar(element) {
+    this.hasScrollbar = element.scrollHeight > element.clientHeight;
+  }
+
+  @action
+  computeResizedEntries(entries) {
+    const element = entries[0].target;
+    this.hasScrollbar = element.scrollHeight > element.clientHeight;
```

Comment:
Can't you just call `this.computeHasScrollbar(element)` here?

Reply:
Yes right I could indeed 👌👌👌